content/support-training.html

```diff
@@ -76,6 +76,12 @@ <h2>Commercial support</h2>
               </a>
           </div>
 
+          <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://envite.de" class="btn btn-training btn-default btn-large" target="_blank">
```

Comment:
We don't require it (because others also don't always have it), but in case you have a Prometheus-specific page you can link to, it would be greatly preferred. That way, anyone looking for support and clicking through all the various support companies will understand much better how envite can help them specifically around Prometheus. Or if you don't have such a page yet, it's also a good idea to update this later with one.

Reply:
At the Moment we don't have one, but we will add it later.